docs/supported-libraries.md

```diff
@@ -48,9 +48,9 @@ These are the supported libraries and frameworks:
-| [Couchbase Client](https://github.com/couchbase/couchbase-java-client)                                                                      | 2.0 - 2.7.x,<br>3.1+               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Database Client Spans]                                                                                                                 |
+| [Cassandra Driver](https://github.com/datastax/java-driver)                                                                                 | 3.0+                               | [opentelemetry-cassandra-4.4](../instrumentation/cassandra/cassandra-4.4/library)                                                                                                                                                                                                                                                                                                       | [Database Client Spans], [Database Client Metrics]                                                                                      |
+| [Clickhouse Client](https://github.com/ClickHouse/clickhouse-java)                                                                          | 0.5+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Database Client Spans], [Database Client Metrics]                                                                                      |
+| [Couchbase Client](https://github.com/couchbase/couchbase-java-client)                                                                      | 2.0 - 2.7.x,<br>3.1+               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [Database Client Spans], [Database Client Metrics]                                                                                      |
```

Comment:
idk if this is worth mentioning here but with couchbase we have 2 instrumentations that generate different telemetry. The one for 3.x that uses tracing built into couchbase does not capture database metrics.